src/System.Globalization/tests/CompareInfo/CompareInfoCompare.cs

```diff
@@ -561,6 +513,18 @@ public void Test148()
         [Fact]
         public void Test156() { Test(CultureInfo.InvariantCulture, new string('a', 5555), new string('a', 5555), 0, CompareOptions.None); }
 
+        [Fact]
+        public void Test157() { Test(CultureInfo.InvariantCulture, "foobar", "FooB\u00C0R", 0, CompareOptions.IgnoreNonSpace | CompareOptions.IgnoreCase); }
+
+        [Fact]
+        public void Test158() { Test(CultureInfo.InvariantCulture, "foobar", "FooB\u00C0R", -1, CompareOptions.IgnoreNonSpace); }
+
+        [Fact]
+        public void Test159() { Test(CultureInfo.InvariantCulture, "\uFF66", "\u30F2", 0, CompareOptions.IgnoreWidth); }
+
+        [Fact]
+        public void Test160() { Test(CultureInfo.InvariantCulture, "\uFF66", "\u30F2", 1, CompareOptions.IgnoreCase); }
```

Comment:
Yes, this was a bad assumption on my part.  It is fixed in the commit I pushed yesterday.
I had the wrong sorting direction for halfwidth and fullwidth characters in our ICU collation rules.  I've changed the product code to match Windows, and updated the test.